pkg/metrics/grafana/tiproxy_summary.jsonnet

```diff
@@ -134,6 +134,12 @@ local cpuP = graphPanel.new(
     'rate(process_cpu_seconds_total{k8s_cluster="$k8s_cluster", tidb_cluster="$tidb_cluster", instance=~"$instance", job="tiproxy"}[1m])',
     legendFormat='{{instance}}',
   )
+)
+.addTarget(
+  prometheus.target(
+    'tiproxy_server_maxprocs{k8s_cluster="$k8s_cluster", tidb_cluster="$tidb_cluster", instance=~"$instance", job="tiproxy"}',
+    legendFormat='limit-{{instance}}',
```

Comment:
What is this?

Reply:
As mentioned in the PR description, the CPU limit.

Reply:
You can also see it in TiDB-Runtime / CPU Usage.